internal/dinosaur/pkg/cmd/central/create.go

```diff
@@ -33,34 +34,29 @@ func NewCreateCommand(env *environments.Env) *cobra.Command {
 	return cmd
 }
 
-func runCreate(env *environments.Env, cmd *cobra.Command, _ []string) {
+func runCreate(client *fleetmanager.Client, cmd *cobra.Command, _ []string) {
 	name := flags.MustGetDefinedString(FlagName, cmd.Flags())
 	region := flags.MustGetDefinedString(FlagRegion, cmd.Flags())
 	provider := flags.MustGetDefinedString(FlagProvider, cmd.Flags())
-	owner := flags.MustGetDefinedString(FlagOwner, cmd.Flags())
-	multiAZ := flags.MustGetBool(FlagMultiAZ, cmd.Flags())
-	clusterID := flags.MustGetDefinedString(FlagClusterID, cmd.Flags())
-	orgID := flags.MustGetDefinedString(FlagOrgID, cmd.Flags())
 
-	var centralService services.DinosaurService
-	env.MustResolveAll(&centralService)
-
-	centralRequest := &dbapi.CentralRequest{
-		Region:         region,
-		ClusterID:      clusterID,
-		CloudProvider:  provider,
-		MultiAZ:        multiAZ,
-		Name:           name,
-		Owner:          owner,
-		OrganisationID: orgID,
+	request := public.CentralRequestPayload{
+		Region:        region,
+		CloudProvider: provider,
+		Name:          name,
+		MultiAz:       true,
 	}
 
-	if err := centralService.RegisterDinosaurJob(centralRequest); err != nil {
-		glog.Fatalf("Unable to create central request: %s", err.Error())
+	const async = true
+	centralRequest, _, err := client.PublicAPI().CreateCentral(cmd.Context(), async, request)
+	if err != nil {
+		glog.Error(err)
+		return
 	}
-	indentedCentralRequest, err := json.MarshalIndent(centralRequest, "", "    ")
+
+	centralJSON, err := json.Marshal(centralRequest)
 	if err != nil {
-		glog.Fatalf("Failed to format central request: %s", err.Error())
+		glog.Errorf(apiErrorMsg, "create", err)
```

Comment:
I think this error message is wrong. It should be 
```suggestion
		glog.Errorf("Failed to marshal Central: %v", err)
```

Reply:
done